demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py

```diff
@@ -1,82 +1,553 @@
 from TestSuite.json_based import JSONBased
 
 
-def test_doc_review_with_release_notes_is_skipped_on_invalid_yml_file(malformed_integration_yml):
+class TestDocReviewFilesAreFound:
+    """
+    Tests scenarios in which files are found before performing doc-review.
     """
-    Given -
-        malformed yml integration file.
 
-    When -
-        Calling doc-review with --release-notes.
+    def test_find_files_from_invalid_path(self):
+        """
+        Given -
+            invalid path file.
+
+        When -
+            trying to find files to do doc-reviews on.
+
+        Then -
+            Ensure no files are found.
+        """
+        doc_review = DocReviewer(file_path='test')
+        assert doc_review.run_doc_review()
+        assert not doc_review.files
+
+    def test_find_files_from_dir(self, valid_spelled_content_pack):
+        """
+        Given -
+            valid pack directory path.
+
+        When -
+            trying to find files to do doc-reviews on.
+
+        Then -
+            Ensure the files that are found exist in the directory.
+        """
+        doc_review = DocReviewer(file_path=valid_spelled_content_pack.path)
+        doc_review.get_files_to_run_on()
+        for file in doc_review.files:
+            assert path.exists(file)
+
+    def test_find_single_file(self, valid_spelled_content_pack):
+        """
+        Given -
+            valid integration yml file path.
+
+        When -
+            trying to find the file to do the doc-review on.
+
+        Then -
+            Ensure the file that was found exist in the directory.
+        """
+        doc_review = DocReviewer(file_path=valid_spelled_content_pack.integrations[0].yml.path)
+        doc_review.get_files_to_run_on()
+        for file in doc_review.files:
+            assert path.exists(file)
+
+    def test_find_files_from_git(self, mocker, valid_spelled_content_pack):
+        """
+        Given -
+            valid pack directory path.
+
+        When -
+            trying to find files to do doc-reviews on from git.
+
+        Then -
+            Ensure the files that git reports are the same found that are meant to be doc-reviewed.
+        """
+        changed_files_mock = [
+            valid_spelled_content_pack.integrations[0].yml.path, valid_spelled_content_pack.scripts[0].yml.path
+        ] + [rn.path for rn in valid_spelled_content_pack.release_notes]
 
-    Then -
-        Ensure that no exception/error is raised and that the malformed files were not added to the files for review.
+        mocker.patch.object(
+            DocReviewer,
+            'gather_all_changed_files',
+            return_value=changed_files_mock
+        )
+        doc_review = DocReviewer(file_path='', use_git=True)
+        doc_review.get_files_to_run_on()
+        assert doc_review.files == set(changed_files_mock)
+
+    def test_find_only_supported_files(self, valid_spelled_content_pack):
```

Comment:
I guess the unsupported files test can be even dropped, because it is actually being checked in ```test_find_only_supported_files``` function when I loop through the files that were found.
Removing ```test_find_unsupported_files``` test